app/src/backend/ResultsService.ts

```diff
@@ -84,3 +93,177 @@ export async function getEmissionResults(inventoryId: string) {
     topEmissionsBySubSector: topSubSectorEmissionsWithPercentage,
   };
 }
+
+interface ActivityForSectorBreakdown {
+  reference_number: string;
+  input_methodology: string;
+  activity_data_jsonb: Record<string, any>;
+  co2eq: string;
+  subsector_name: string;
+  scope_name: string;
+}
+
+interface UngroupedActivityData {
+  activityTitle: string;
+  activityValue: string;
+  activityUnits: string;
+  activityEmissions: bigint;
+  emissionsPercentage: number;
+  subsectorName: string;
+  scopeName: string;
+}
+
+interface GroupedActivity {
+  activityValue: string;
+  activityUnits: string;
+  totalActivityEmissions: string; // Using string to avoid jest's "Don't know how to serialize Bigint" error
+  totalEmissionsPercentage: number;
+}
+
+const getActivitiesForSectorBreakdown = async (
+  inventoryId: string,
+  sectorName: string,
+): Promise<ActivityForSectorBreakdown[]> => {
+  const rawQuery = `
+        SELECT av.activity_data_jsonb, sc.reference_number, input_methodology, av.co2eq, s.sector_name, ss.subsector_name, scope.scope_name
+        FROM "ActivityValue" av
+            JOIN "InventoryValue" iv ON av.inventory_value_id = iv.id
+            JOIN "Sector" s ON iv.sector_id = s.sector_id
+            JOIN "SubSector" ss ON iv.sub_sector_id = ss.subsector_id
+            JOIN "SubCategory" sc ON iv.sub_category_id = sc.subcategory_id
+            JOIN "Scope" scope ON scope.scope_id = sc.scope_id OR ss.scope_id = scope.scope_id
+        WHERE inventory_id = :inventoryId
+            AND s.sector_name ilike :sectorName
+    `;
+
+  try {
+    return (await db.sequelize!.query(rawQuery, {
+      replacements: { inventoryId, sectorName: sectorName.replace("-", " ") },
```

Comment:
Why are you doing this here?
These are probably translation strings that should be translated in the frontend, not made to look like english strings?

Reply:
if we use the translation, we might receive it in spanish, portuguese or german. so I'm planning on sending the translation string directly (eg: stationary-energy), and then I have to remove the "-" so query in the db